package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "daedalus",
   "productName": "Daedalus",
-  "version": "2.3.0-FC2",
+  "version": "2.1.0",
```

Comment:
@daniloprates please revert this change. The version should remain 2.3.0-FC2...

Reply:
@nikolaglumac this is only for [testing purposes](https://github.com/input-output-hk/daedalus/pull/2195/commits/03ff1637d59f92e6107d86c8f09ed06373ea4552).